haiku-apps/commandtimer/commandtimer-2.0~git.recipe

```diff
@@ -0,0 +1,48 @@
+SUMMARY="Run a command at a after a specified ammount a time has passed"
```

Comment:
"Run a command at a after a specified ammount a time has passed" isn't correct english, and it's "amount"